tt_llk_wormhole_b0/common/inc/sfpu/ckernel_sfpu_binary_bitwise.h

```diff
@@ -19,16 +23,17 @@ enum class BinaryBitwiseOp : uint8_t
     XOR = 2,
 };
 
-template <bool APPROXIMATION_MODE, BinaryBitwiseOp BITWISE_OP, int ITERATIONS = 8>
+template <bool APPROXIMATION_MODE, BinaryBitwiseOp BITWISE_OP, InstrModLoadStore INSTRUCTION_MODE = INT32, int ITERATIONS = 8>
 inline void _calculate_sfpu_binary_bitwise_(const uint dst_offset)
 {
+    constexpr auto ins_mode = static_cast<std::underlying_type_t<DataFormat>>(INSTRUCTION_MODE);
```

Comment:
And here too, it should be InstrModLoadStore instead of DataFormat:
```suggestion
    constexpr auto instruction_mode = static_cast<std::underlying_type_t<InstrModLoadStore >>(INSTRUCTION_MODE);
```